defs.bzl

```diff
@@ -0,0 +1,23 @@
+def gmaven_artifact(fqn):
+  parts = fqn.split(":")
+  packaging = "aar"
```

Comment:
The default should probably be jar. Every other tool that parses maven coordinates seems to think so.

Reply:
I just realized why you did this: Because most of them are aar's and a user doesn't necessarily know that.

Perhaps a good solution here would be: In addition to generating the maven_aar/maven_jar (which in turn generate a package named "jar" or "aar"), also generate an alias rule somewhere* with a name that does not depend on "aar" or "jar", so then the BUILD file writer does not need to know which it is.

* I'm not sure where that somewhere would be. Maybe the maven_jar/maven_aar rule could create the alias in it's repository? Otherwise we'd need to create a new repository with these aliases, which is ugly because it adds yet another layer of abstraction. We could use bind, but that I think that might be deprecated.

Reply:
I've switched it to use jar by default. I like the alias idea, but I think that's out of scope of this PR.